gnd/src/main/java/com/google/android/gnd/persistence/local/room/entity/FeatureMutationEntity.java

```diff
@@ -79,6 +81,7 @@ public static FeatureMutationEntity fromMutation(FeatureMutation m) {
         .setFeatureId(m.getFeatureId())
         .setLayerId(m.getLayerId())
         .setNewLocation(m.getNewLocation().map(Coordinates::fromPoint).orElse(null))
+        .setNewPolygonVertices(listToString(m.getNewPolygonVertices()))
```

Comment:
Same comment re naming of `listToString` and `stringToList`.

Reply:
updated